### PR TITLE
Change Bower to NPM.

### PR DESCRIPTION
**Pull request by snickom**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #35

### CODE REVIEW

**Comment by tomlutzenberger** [comment]
@samdark What do you think? Will there be a switch to npm in the other repos?

**Comment by schmunk42** [comment]
I think switching to npm makes sense for this package.

File: composer.json [hunk 1]
```diff
@@ -37,7 +37,8 @@
     ],
     "require": {
         "yiisoft/yii2": "~2.0|~2.1@dev|~3.0@dev",
-        "bower-asset/bootstrap": "~4.0"
+        "npm-asset/bootstrap": "~4.1",
+        "npm-asset/popper.js": "~1.14.3"
```
[the remark this answers is not in the document]
Comment (cebe) [reply composer.json]: This is the same as we have with jquery here:

https://github.com/yiisoft/yii2/blob/558cd482d8a0525fceec8646ea20ccc9b7c3fdf4/framework/composer.json#L73

**Comment by machour** [comment]
Hi @snickom,

We'd like to publish the first release of this package soon. Could you take care of fixing conflicts and applying the suggested change to `composer.json` ?

**Comment by snickom** [comment]
I make another pull request (#85) with fixed composer.json, because I lost my old forked repository, sorry.

**Pull request closed without merging**